src/components/organisms/tracker-charts.tsx

```diff
@@ -220,15 +206,31 @@ export const TrackerCharts: FC<TrackerChartsProps> = ({
         </>
       )}
       {chartDataIsAvailable(positivesData) && (
-        <Card padding={{h: 12}}>
-          <TrackerBarChart
-            title={t('charts:positiveTests:title')}
-            description={t('charts:positiveTests:hint')}
-            axisData={positivesData.axisData}
-            chartData={positivesData.chartData}
-            averagesData={positivesData.averagesData}
-          />
-        </Card>
+        <>
+          <Card padding={{h: 12}}>
+            <TrackerBarChart
+              title={t('charts:positiveTests:title')}
+              description={t('charts:positiveTests:hint')}
+              axisData={positivesData.axisData}
+              chartData={positivesData.chartData}
+              averagesData={positivesData.averagesData}
+            />
+          </Card>
+          <Spacing s={20} />
+        </>
+      )}
+      {chartDataIsAvailable(testsData) && (
+        <>
```

Comment:
There's only one component at the top level here, no siblings, so the fragment isn't needed